message.go

```diff
@@ -17,129 +14,40 @@ type Message struct {
 	Tags     []string       `json:"tags,omitempty"`     // List of tags that may or not map to emojis
 	Priority Priority       `json:"priority,omitempty"` // Message priority with 1=min, 3=default and 5=max
 	Actions  []ActionButton `json:"actions,omitempty"`  // Custom user action buttons for notifications
-	ClickURL *url.URL       `json:"click,omitempty"`    // Website opened when notification is clicked
-	IconURL  *url.URL       `json:"icon,omitempty"`     // URL to use as notification icon
-	Delay    time.Duration  `json:"delay,omitempty"`    // Duration to delay delivery
+	ClickURL string         `json:"click,omitempty"`    // Website opened when notification is clicked
+	IconURL  string         `json:"icon,omitempty"`     // URL to use as notification icon
+	Delay    interface{}    `json:"delay,omitempty"`    // Duration to delay delivery (string, int, whatever)
```

Comment:
I don't know if this is the best route to take. I would rather keep typing strong here and use a duration. If you have code that wants to pass this as a humanized time it should probably be getting parsed as a valid duration prior and put in this struct